yarn.lock

```diff
@@ -11788,7 +11788,7 @@ preact@^10.5.13:
   resolved "https://registry.yarnpkg.com/preact/-/preact-10.5.15.tgz#6df94d8afecf3f9e10a742fd8c362ddab464225f"
   integrity sha512-5chK29n6QcJc3m1lVrKQSQ+V7K1Gb8HeQY6FViQ5AxCAEGu3DaHffWNDkC9+miZgsLvbvU9rxbV1qinGHMHzqA==
 
-prebid.js@guardian/prebid.js#2e3b96d:
+"prebid.js@github:guardian/prebid.js#2e3b96d":
```

Comment:
come on bot you can do better!

Reply:
Could the bot fix yarn.lock if required?

Reply:
Is there a command to fix it? Or would it need to edit the file

Reply:
Yeah I was thinking just something like sed and a git commit. A bit of a sticky plaster but in lieu of better options